lua/telescope/operators.lua

```diff
@@ -0,0 +1,23 @@
+local operators = {}
+
+local last_operator = { callback = function(_) end, opts = {} }
+
+--- Execute the last saved operator callback and options
+operators.operator_callback = function()
+  last_operator.callback(last_operator.opts)
+end
+
+--- Enters operator-pending mode, then executes callback.
+--- See `:h map-operator`
+---
+---@param callback function: the function to call after exiting operator-pending
+---@param opts table: options to pass to the callback
+operators.run_operator = function(callback, opts)
+  last_operator = { callback = callback, opts = opts }
+  vim.o.operatorfunc = "v:lua.require'telescope.operators'.operator_callback"
+  -- feed g@ to enter operator-pending mode
+  -- 'i' required for which-key compatibility, etc.
+  vim.api.nvim_feedkeys("g@", "mi", false)
```

Comment:
do you think after running this line we should clear the operator? Or perhaps on line 7? This way we drop the reference to the picker and it can be garbage collected later, which seems like a good idea.

Maybe it should be:

```lua
operators.operator_callback = function()
  last_operator.callback(last_operator.opts)
  last_operator = { callback = function(_) end, opts = {} }
end

Reply:
Right, it's only a reference to the most recent picker/callback function used with `operators.run_operator`. At least for this picker it would just keep a reference to `git.bcommits_range` and its opts.